src/components/ErrorBoundary.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import Alert from "@material-ui/lab/Alert";
+
+export function defaultFormatter(error) {
```

Comment:
```suggestion
function defaultFormatter(error) {
```

Let's _not_ export this